dxfeed/wrappers/DXFeed.py

```diff
@@ -0,0 +1,54 @@
+from dxfeed.core.DXFeedPy import *
+from dxfeed.wrappers.DXFeedSubscription import DXFeedSubscription
+
+
+class DXFeed(object):
+    """
+    Class for connection management. After successful creation Instance will be connected to server
+    with provided credentials
+
+    Attributes
+    ----------
+    connection_address: str
+        One of possible connection addresses:
+        
+        - the single address: `host:port` or just `host`
+        - address with credentials: `host:port[username=xxx,password=yyy]`
+        - multiple addresses: `(host1:port1)(host2)(host3:port3[username=xxx,password=yyy])`
+        
+        Default: demo.dxfeed.com:7300
+    """
+    def __init__(self, connection_address: str = 'demo.dxfeed.com:7300'):
+        self.__con_address = connection_address
+        self.__connection = ConnectionClass()
+        self.connect()
+
+    @property
+    def connection_status(self):
+        return dxf_get_current_connection_status(self.__connection, return_str=True)
+
+    @property
+    def address(self):
+        return dxf_get_current_connected_address(self.__connection)
+
+    def connect(self):
+        if dxf_get_current_connection_status(self.__connection, return_str=False):
+            dxf_close_connection(self.__connection)
+        self.__connection = dxf_create_connection(self.__con_address)
+        return self
+
+    def create_subscription(self, event_type: str, data_len: int = 100000, date_time: Union[str, datetime] = None):
```

Comment:
Do we really want to have two possible variants of `Subscription` creation? Mb let's have only one possible way to create connection to make it easy to use and handle in future?

Reply:
What is the 2nd method? Creation of Subscription instance itself? I can hide Subscription from import suggestions with `__all__` variable. Is it ok?

Reply:
https://github.com/dxFeed/dxfeed-python-api/commit/653b66eb4a2f833e00f33403a2a51641612673ff